### PR TITLE
chore: update pkgs/tools (Go 1.19.4, containerd 1.6.11)

### PR DESCRIPTION
**Pull request by smira**
Update to the latest pkgs/tools to fix the build due to vulncheck.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

**Review by DmitriyMV — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -34,7 +34,7 @@ require (
 	github.com/beevik/ntp v0.3.0
 	github.com/cenkalti/backoff/v4 v4.2.0
 	github.com/containerd/cgroups v1.0.4
-	github.com/containerd/containerd v1.6.10
+	github.com/containerd/containerd v1.6.11
```
Comment (frezbo) [inline go.mod:37]: 1.6.12 has the cve fix

**Review by frezbo — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (frezbo) [reply go.mod:37]: i think we can update pkgs and then merge this

Comment (smira) [reply go.mod:37]: yes, but we don't have it in pkgs yet

File: pkg/machinery/constants/constants.go [hunk 2]
```diff
@@ -423,7 +423,7 @@ const (
 	TrustdUserID = 51
 
 	// DefaultContainerdVersion is the default container runtime version.
-	DefaultContainerdVersion = "1.6.10"
+	DefaultContainerdVersion = "1.6.11"
```
[the remark this answers is not in the document]
Comment (smira) [reply pkg/machinery/constants/constants.go:426]: no, this is .11 as it is in pkgs right now

**Pull request merged**